clang/lib/StaticAnalyzer/Checkers/WebKit/RawPtrRefMemberChecker.cpp

```diff
@@ -192,7 +202,8 @@ class RawPtrRefMemberChecker
 
     const auto Kind = RD->getTagKind();
     // FIMXE: Should we check union members too?
-    if (Kind != TagTypeKind::Struct && Kind != TagTypeKind::Class)
+    if (Kind != TagTypeKind::Struct && Kind != TagTypeKind::Class &&
```

Comment:
Can the FIXME comment be removed?

Reply:
Oh oops, I meant to remove that and forgot! Thanks for spotting it!